packages/conf-mingw-w64-gmp-i686/conf-mingw-w64-gmp-i686.1/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+synopsis: "Virtual package for libgmp on i686 mingw-w64 (32-bit x86)"
+description: "Ensures the i686 version of libgmp for the mingw-w64 project is available"
+maintainer: "David Allsopp <david@tarides.com>"
+license: "GPL-1.0-or-later"
+homepage: "http://gmplib.org/"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+flags: conf
+available: os = "win32"
+build: ["pkgconf" "--personality=i686-w64-mingw32" {os-distribution = "cygwin"} "libgmp"]
```

Comment:
Shouldn't you use `gmp` instead of `libgmp` as an argument for `pkgconf`?

Reply:
fixed